packages/ui/src/lib/adapters/zkitter-god-mode/index.ts

```diff
@@ -0,0 +1,210 @@
+import { getGlobalAnonymousFeed, getProvider } from '$lib/services'
+import { sleep } from '$lib/utils'
+import { GroupAdapter } from '../zkitter/group-adapter'
+import type { Signer } from 'ethers'
+import { posts } from '$lib/stores/post'
+import { RELAYER_URL } from '../../constants'
+import { ZkitterAdapter } from '../zkitter'
+import { tokens } from '$lib/stores/tokens'
+import { generateRLNProofForNewPersona, prover } from '../zkitter/utils'
+import { UserState } from '@unirep/core'
+import type { DraftPersona } from '$lib/stores/persona'
+
+const epochDuration = 8 * 60 * 60 * 1000
+
+export class ZkitterAdapterGodMode extends ZkitterAdapter {
+	constructor() {
+		super()
+	}
+
+	start(): Promise<void> {
+		tokens.set({
+			repStaked: 0,
+			repTotal: Infinity,
+			go: Infinity,
+			loading: false,
+			epochDuration,
+			timeToEpoch: 0,
+		})
+		return super.start()
+	}
+
+	async publishPost(
+		personaId: string,
+		text: string,
+		images: string[],
+		signer: Signer,
+	): Promise<string> {
+		const { Post, MessageType, PostMessageSubType } = await import('zkitter-js')
+
+		// User did not join the persona yet
+		if (await this.queryPersonaJoined(personaId)) {
```

Comment:
this should be checking the negative of `await this.queryPersonaJoined(personaId)` instead